samples/performance/README.md

```diff
@@ -0,0 +1,78 @@
+# Gateway Performance Measurement
+
+Measure performance of the Gateway SDK.
+
+## Overview
```

Comment:
Overall Suggestion: 
1-Remove this from Samples to tests folder. This should be considered as a test; 
2-Instead of creating Requirement, Create a Test Plan; I know this is V1, but this is a test. Even if you are creating new modules. These are just test artifacts in IMO; 
3-On the TEst Plan I suggest to include more scenarios. The scenarios you started with are great, however I would suggest: 
    3.1 - Instrumentation of current samples; 
    3.2 - Adding a Scenarios of Multiple Simulator for 1 metric; 
    3.3 - One Simulator, Multiple Metrics; 
    3.4 - Maximum amount of simulator for 1 metric that you can create; 

That just for starters. 

Great job David.